src/components/console/graphs/graph-empty.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export const GraphEmpty: React.FC<GraphEmptyProps> = ({ height = 180, loading = false }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      style={{
```

Comment:
Should we put this style in a separate file?

Reply:
Everything under the `console/` directory is copied from the console codebase, so I'm not inclined to do any tidying up of that code here. Over time, everything under `console/` should be refactored anyway.